api/server/runner.go

```diff
@@ -208,7 +207,7 @@ func (s *Server) serve(ctx context.Context, c *gin.Context, appName string, foun
 		c.JSON(http.StatusAccepted, map[string]string{"call_id": task.ID})
 
 	default:
-		result, err := runner.RunTask(s.tasks, ctx, cfg)
+		result, err := Api.Runner.RunTask(ctx, cfg)
```

Comment:
Another consequence of this change. We are now relying even more on something that we are trying to avoid: the `Api` global variable. https://github.com/iron-io/functions/blob/master/api/server/server.go#L23-L25

```
// Would be nice to not have this is a global, but hard to pass things around to the
// handlers in Gin without it.
```

Is this the path we want to be going to?